docs/get-started/getting-started-gcp.md

```diff
@@ -16,6 +23,108 @@ parent: Getting-Started
   GPUs](#submit-spark-jobs-to-a-dataproc-cluster-accelerated-by-gpus)
 * [Build custom Dataproc image to accelerate cluster initialization time](#build-custom-dataproc-image-to-accelerate-cluster-init-time)
 
+## Quick Start Prerequisites
+
+* gcloud CLI is installed: https://cloud.google.com/sdk/docs/install
+* python 3.8+
+* `pip install spark-rapids-user-tools`
+
+## Qualify CPU Workloads for GPU Acceleration
+
+The [qualification tool](https://nvidia.github.io/spark-rapids/docs/spark-qualification-tool.html) is launched on a Dataproc cluster that has applications that have already run.
+The tool will output the applications recommended for acceleration along with estimated speed-up
+and cost saving metrics.  Additionally, it will provide information on how to launch a GPU-
+accelerated cluster to take advantage of the speed-up and cost savings.
+
+Usage: `spark_rapids_dataproc qualification --cluster <cluster-name> --region <region>`
```

Comment:
Should we mention the options that this can be locally installed or on the cluster or any Google Compute instance?

Reply:
The user tools need to be run outside the cluster.  It won't work outside the cluster.